proto/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Generate client code (go & json) from API protocol buffers
-FROM golang:1.17.8 as generator
+FROM --platform=linux/amd64 golang:1.17.8 as generator
```

Comment:
This is needed for building locally on Apple Silicon, since omitting it will produce an image for arm64 and will result in protoc failing to run.